specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -237,21 +237,22 @@ model JailbreakAnalysisResult {
   detected: boolean;
 }
 
-
 #suppress "@azure-tools/typespec-azure-core/documentation-required" "MUST fix in next update"
 @added(ContentSafety.Versions.v2024_02_15_Preview)
 @doc("Domain.")
-enum Domain {
-  Generic,
-  Medical,
+union Domain {
+  string,
+  Generic: "Generic",
+  Medical: "Medical",
 }
 
 #suppress "@azure-tools/typespec-azure-core/documentation-required" "MUST fix in next update"
 @added(ContentSafety.Versions.v2024_02_15_Preview)
 @doc("Task type.")
-enum Task {
-  Summarization,
-  QnA,
+union Task {
```

Comment:
This name conflicts with some languages' types/keywords and will never pass SDK archboard. Can you elaborate? Better to have the REST API and SDKs match as much as possible, or customers will be confused about the mapping.